src/line-chart/line-chart.component.ts

```diff
@@ -175,7 +178,6 @@ export class LineChartComponent extends BaseChartComponent {
   @Input() yAxisTickFormatting: any;
   @Input() roundDomains: boolean = false;
   @Input() tooltipDisabled: boolean = false;
-  @Input() showSeriesOnHover: boolean = true;
```

Comment:
We may revisit that in the future. The performance impact they had was too big to ignore, so we had to remove them for now.